<launch>

  <node name="qsr_lib" pkg="qsr_lib" type="qsrlib_ros_server.py" output="screen" respawn="true"/>
  <node name="goal_pose_republisher" pkg="hrsi_representation" type="goal_pose_republisher.py" output="screen" respawn="true"/>
  <node name="people_tracker" pkg="people_tracker_emulator" type="posestamped_to_ppl_tracker_msgs.py" output="screen" respawn="true">
    <param name="people" type="string" value="/people"/>
  </node>
  <node pkg="hrsi_representation" type="online_qtc_creator.py" name="online_qtc_creator" output="screen" respawn="true">
    <param name="ppl_topic" type="string" value="/people_tracker/positions"/>
    <param name="processing_rate" type="int" value="60"/>
    <param name="max_buffer_size" type="int" value="10"/>
    <param name="decay_time" type="double" value="20."/>
    <param name="quantisation_factor" type="double" value="0.01"/>
    <param name="distance_threshold" type="double" value="6.0"/>
    <param name="smoothing_rate" type="double" value="0.3"/>
    <param name="qtc_type" type="int" value="2"/>
    <param name="no_collapse" type="bool" value="false"/>
    <param name="prune_buffer" type="bool" value="false"/>
    <param name="validate" type="bool" value="true"/>
  </node>
  <node pkg="hrsi_state_prediction" type="state_predictor.py" name="qtc_state_predictor" output="screen" respawn="true"/>
  <node pkg="hrsi_velocity_costmaps" type="velocity_costmap_server.py" name="velocity_costmap_server" output="screen" respawn="true"/>

</launch>
